### DISCUSSION TITLE
Search PRs reviewed by team

### DISCUSSION THREAD

**Discussion by iamricard** [community/community#12503] [Pull Requests]
Hi 👋🏼 thanks for the awesome work yinz are doing with improving the pull requests product!

In my org we've been relying a lot on labels to filter and search pull requests, but sometimes we forget to add them (because we forgot to configure the automated labeler, for example). Our main use-case for labels, though, it to somewhat track ownership. We already do that through the `CODEOWNERS` file. The problem we've found with just using `CODEOWNERS` is that there's no way for us to search for pull requests that have been reviewed by a team.

Today, we can search by `team-review-requested`, as documented [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests#filtering-issues-and-pull-requests). However, as soon as a someone from that team reviews the pull request, it disappears from the search results (which makes sense since the team disappears from "reviewers". What we'd like is for us to be able to query for something like this:

* `team-review-requested:DataDog/my-team -team-reviewed-by:DataDog/my-team` - this would yield all PRs that requested a review from `team-review-requested:DataDog/my-team` and haven't gotten it yet (from that team)
* `team-review-requested:DataDog/my-team` - this would yield all PRs that requested a review from `team-review-requested:DataDog/my-team`, regardless of whether or not they got a review from that team

**Comment by iamricard** [comment]
Bump @willsmythe, I've seen you comment in other PR discussions 👀 Wondering if this is on y'alls roadmap.

**Comment by ADalcherPS** [comment]
My team is running into this as well. It would be great if we could get this onto some sort of roadmap
